docs/en/01_Working_with_projects/01_Setting_up_your_project.md

```diff
@@ -70,7 +70,7 @@ For example a `.gitignore` for a CWP recipe codebase might include:
     /cwp-core
     ...
```

Comment:
Could you update this to look more like this: https://github.com/silverstripe/cwp-installer/blob/master/.gitignore